src/order/chain.lean

```diff
@@ -0,0 +1,222 @@
+/-
+Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl
+-/
+import data.set.pairwise
+
+/-!
+# Chains
+
+This file defines chains for an arbitrary relation and proves several formulations of Zorn's Lemma,
```

Comment:
Are there any formulations of Zorn's lemma in this file still, or are they in `order.zorn` now?

Reply:
Whoops, forgot this bit.